packages/device_info/lib/device_info.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
```

Comment:
Should I create two classes, one for Android and one for iOS, or should I have one class with two platform-specific methods?

Option 1:

```dart
class AndroidInfo {
  AndroidOSBuildInfo get info;
}

class IosInfo {
  IosDeviceInfo get info;
}
```

Option 2:

```dart
class DeviceInfo {
  AndroidOSBuildInfo get info;
  IosDeviceInfo get info;
}
```

Or something else? What say ye?